### PR TITLE
[E-node < T499] Implement node.relationshipsExist

### PR DESCRIPTION
**Pull request by mpintaric55334**
### Description

Please briefly explain the changes you made here.

### Pull request type

- [ ] Bugfix
- [ ] Algorithm/Module
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### Related issues

Delete if this PR doesn't resolve any issues. Link the issue if it does.

######################################

#### Module/Algorithm
- [ ] Core algorithm/module implementation
- [ ] [Query module](https://memgraph.com/docs/memgraph/reference-guide/query-modules) implementation
- [ ] Unit tests
- [ ] End-to-end tests
- [ ] Code documentation
- [ ] README short description
- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)
- [ ] Update GQLALchemy signatures in [query builder](https://github.com/memgraph/gqlalchemy/blob/main/gqlalchemy/graph_algorithms/query_builder.py  ) using [query module signature generator](https://github.com/memgraph/gqlalchemy/blob/main/scripts/query_module_signature_generator.py)

######################################

### CODE REVIEW

File: cpp/node_module/algorithm/node.hpp [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+#include <string>
 
-namespace Node {}  // namespace Node
+namespace Node {
+/*relationships_exist constants*/
+constexpr std::string_view kProcedureRelationshipsExist = "relationships_exist";
+constexpr std::string_view kReturnRelationshipsExist = "result";
+constexpr std::string_view kArgumentNodesRelationshipsExist = "node";
+constexpr std::string_view kArgumentRelationshipsRelationshipsExist = "relationships";
+void RelationshipsExist(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
```
Comment (Josipmrden) [inline cpp/node_module/algorithm/node.hpp:12]: empty line after `constexpr`

File: cpp/node_module/algorithm/node.hpp [hunk 2]
```diff
@@ -1,5 +1,14 @@
 #pragma once
 
 #include <mgp.hpp>
+#include <string>
 
-namespace Node {}  // namespace Node
+namespace Node {
+/*relationships_exist constants*/
```
Comment (Josipmrden) [inline cpp/node_module/algorithm/node.hpp:7]: empty line after `namespace Node`

File: cpp/node_module/algorithm/node.cpp [hunk 3]
```diff
@@ -1 +1,53 @@
 #include "node.hpp"
+
+bool Node::RelationshipExist(const mgp::Node &node, std::string &rel_type) {
+  char direction{' '};
+  if (rel_type[0] == '<' && rel_type[rel_type.size() - 1] == '>') {
+    throw mgp::ValueException("Invalid relationship specification!");
+  }else if (rel_type[rel_type.size() - 1] == '>') {
```
Comment (Josipmrden) [inline cpp/node_module/algorithm/node.cpp:7]: split } and else if with one space

**Review by antepusic — APPROVED** [review]
Think we're good to go :rocket:

**Pull request merged**